enterprise-cfengine-guide/settings.markdown

```diff
@@ -66,12 +66,26 @@ will not see any hosts in Mission Portal.
 Users without a role will not be able to see any hosts in Mission
 Portal.
 
-**Default Roles:**
+**Preset Roles:**
 
 * admin - The admin role can see everything and do anything.
 * cf_remoteagent - This role allows execution of cf-runagent. It can
   be used from within Design Center to troubleshoot hosts that have
   failed sketch activations.
+  
+**Default Role:**
+
+On `Role Management` page you can set default role by clicking on checkbox located 
```

Comment:
@michaelclelland might have some wording suggestions.

I might say "To set a default role navigate to the `Role Management` page and checking the boxes in the `Default role` column which you would like to apply to all new users."